src/sendbuf.c

```diff
@@ -182,6 +182,9 @@ void ExpandMacro(void) {
 	    early_started = 0;
 //                              sending_call = 0;
 	}
+	if (cqmode == CQ && resend_call != RESEND_NOT_SET) {
+	    strcpy(sentcall, hiscall);
```

Comment:
`sentcall` is always set, even when sending F9 (question mark). This could be seen both as a bug or a feature. E.g. one could configure a "suppress resend" key by defining an F-key empty.

Reply:
> This could be seen both as a bug or a feature.

For what should we consider it? :)

> even when sending F9 (question mark) ... a "suppress resend" key by defining an F-key empty.

If the OP presses `F9`, and sends a `?`, then the station will answer with his call again. Consider the OP records the correct call, in this case Tlf sends it, so `sentcall` will be overwritten, then will equal with `hiscall`, so the condition in [logit.c](https://github.com/airween/tlf/blob/resendhiscall/src/logit.c#L171) won't match. But, if the OP records the wrong call again, and station corrects it again, OP fixes it, then finally Tlf sends the correction.

So, the last sent `hiscall` always copied into `sentcall`, and if it matches with `hiscall` at the end, then `RESEND` feature won't triggered. I think that's the point of this feature. But please correct me if I'm wrong.

Reply:
F9 was just an example of an F-key not actually related to sending their call.
I was just saying that `hiscall` is always stored and this could have unexpected effects. But if it is by design, then it's also fine.

Reply:
Right. Let's leave it that way, and see the experiences.

If you agree, please resolve this conversation.